.eslintrc.js

```diff
@@ -0,0 +1,175 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    jest: true,
+  },
+  globals: {
+    React: 'writable',
+  },
+  extends: [
+    'plugin:react/recommended',
+    'plugin:react-hooks/recommended',
+    'airbnb',
+    'prettier',
+  ],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 12,
+    sourceType: 'module',
+  },
+  plugins: ['react', '@typescript-eslint', 'prettier'],
+  overrides: [
+    {
+      files: ['*.tsx', '*.ts'],
+      processor: '@graphql-eslint/graphql',
+    },
+    {
+      files: ['*.graphql'],
+      extends: ['plugin:@graphql-eslint/operations-recommended'],
+      rules: {
+        '@graphql-eslint/naming-convention': 'off',
+        '@graphql-eslint/executable-definitions': 'off',
+        // TODO turn these back on
+        '@graphql-eslint/no-unused-fragments': 'warn',
+      },
+      parserOptions: {
+        operations: ['./**/*.tsx', './**/*.ts'],
+        schema: './schema/schemaClient.graphql',
+      },
+    },
+  ],
+  rules: {
+    // TODO turn these back on:
+    'react/no-unstable-nested-components': 'warn',
+    'react/no-unused-class-component-methods': 'warn',
+    'no-unsafe-optional-chaining': 'warn',
+    'react/no-invalid-html-attribute': 'warn',
+    'react/jsx-no-constructed-context-values': 'warn',
+    'react/jsx-no-bind': 'warn',
+
+    // Don't use these libraries, use local wrappers instead
+    'no-restricted-imports': [
+      'error',
+      {
+        name: 'react-toastify',
+        message: 'Please use "components/dls/Toast/toast" instead',
+      },
+      {
+        name: '@apollo/client',
+        importNames: ['useQuery', 'useLazyQuery'],
+        message: 'Please use "gql/queries" instead',
+      },
+    ],
+
+    // note you must disable the base rule as it can report incorrect errors
+    'no-unused-vars': 'off',
+    '@typescript-eslint/no-unused-vars': ['error'],
+
+    // Turning this off because next/link expects an invalid anchor
+    'jsx-a11y/anchor-is-valid': 'off',
```

Comment:
Are these relevant to this repo?

Reply:
No idea how that got in...

Reply:
`copypasta`

Reply:
Oh... it wasn't me... I won't name names though...

Reply:
Wild, they don't seem to be in the file any more. WTH